### ISSUE TITLE
R (Journal) logo and alt text

### ISSUE THREAD

**Issue by hturner** [rjournal/rjournal.github.io#79]
Comment from @ajrgodfrey

> The alt text for the R logo is currently "Logo". This should be "R logo"

The R Journal used to have "The [R logo] Journal" on its front page, wouldn't it be better to use this as the R Journal logo? So replace "[R logo]  The R Journal" with "The [R logo] Journal" as an image, with "The R Journal" as alt text?

**Comment by mitchelloharawild** [comment]
The R Journal used to look like this.

![image](https://github.com/rjournal/rjournal.github.io/assets/16127127/ec9324a8-1021-48b4-9707-02ca688467b5)
![image](https://github.com/rjournal/rjournal.github.io/assets/16127127/16df1775-ff52-4ece-b2f3-ce60fe18f818)

I definitely agree the alt text should be improved.
As for the placement of the R logo (and a potential logo for the R Journal), I think that's a different discussion.
